deploy/internal/statefulset-db.yaml

```diff
@@ -20,25 +20,6 @@ spec:
     spec:
       serviceAccountName: noobaa-db
       terminationGracePeriodSeconds: 60
-      initContainers:
-      #----------------#
-      # INIT CONTAINER #
-      #----------------#
-      - name: init
-        image: NOOBAA_CORE_IMAGE
-        command:
-        - /noobaa_init_files/noobaa_init.sh
-        - init_mongo
```

Comment:
In case we don't use the `init_mongo` argument (and the function it calls `prepare_mongo_pv` iniside `noobaa_init.sh` file) - maybe edit it in this PR.

Reply:
I am not sure if I completely understand.

Reply:
Do we still use `init_mongo` function somewhere?

Reply:
No, we don't. If this PR gets in, I will remove it from the core as well.